python/lsst/ip/isr/measureCrosstalk.py

```diff
@@ -165,20 +167,36 @@ def parseAndRun(cls, *args, **kwargs):
             pickle.dump(resultList, open(results.parsedCmd.dumpRatios, "wb"))
         coeff, coeffErr, coeffNum = task.reduce(resultList)
 
+        calib = CrosstalkCalib()
+        provenance = IsrProvenance()
+
+        calib.coeffs = coeff
+        calib.coeffErr = coeffErr
+        calib.coeffNum = coeffNum
+
         outputFileName = results.parsedCmd.outputFileName
         if outputFileName is not None:
             butler = results.parsedCmd.butler
             dataId = results.parsedCmd.id.idList[0]
-            dataId["detector"] = butler.queryMetadata("raw", ["detector"], dataId)[0]
 
+            # Rework to use new crosstalk calib.
```

Comment:
Please edit comment so it's a bit less wrong in N years when this isn't "new" anymore.